### PR TITLE
WIP_Create 2024_unitaryhack.md

### PR DESCRIPTION
**Pull request by KallieFerguson**
Announcement Blog

@nathanshammah here is a draft we can edit! I want to double check whether we should be adding more sponsors so the list is accurate before it goes live.

### CODE REVIEW

**Comment by KallieFerguson** [comment]
@nathanshammah I can't find where it's going wrong and won't deploy properly!

**Comment by beecastanon** [comment]
@KallieFerguson looks great - just one typo in the last sentence where we should swap 'and' for a comma: "For more information, read our hacker guide and check out ..." should be "For more information, read our hacker guide, check out..."

If you can make that change today I'll push live.

File: src/content/blog/2024_unitaryhack.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- Hackers support existing quantum computing projects. 
+- Hackers get paid for their work while building professional skills such as working on open source projects, and contributing to create a more functional, and featured, quantum computing stack.
+
+For more information, read our hacker guide and check out what happened in [unitaryHACK 2023](https://unitary.fund/posts/unitaryhack_wrap/) and get connected on our [Discord server](http://discord.unitary.fund)!
+
+Make sure you [register](https://unitaryhack.dev/register/) to be eligible to claim your bounties!
+
+**Ready to get hacking? Check out all of our [live bounties](https://unitaryhack.dev/bounties/)!**
+
```
Comment (nathanshammah) [inline src/content/blog/2024_unitaryhack.md:19]:
```suggestion

## Hacking IRL
This is the first year our amazing global community is hosting in-person meetups at various locations, so if you're somewhere near, you can attend:

- May 31: Aalto University CS in Helsinki, Finland, starting at 18:00 for 24 hours. Pizza available!
- May 31: Auditorio Raúl J. Marsal, Ciudad Universitaria, UNAM in Mexico City, Mexico, at 16 HRS UTC-6!
- June 11: University of Washington HUB 145, Seattle, WA, USA, from 9 am to 5 pm, food will be available! 

If you'd like to host an event at your university, drop us a line at hack@unitary.fund, and we'll spread the word! 
```

File: src/content/blog/2024_unitaryhack.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+month: 5
+year: 2024
+---
+
+The Unitary Fund hackathon supporting quantum open source projects is happening now! [unitaryHACK](https://unitaryhack.dev/) is a virtual event encouraging people to make contributions to the open source quantum ecosystem. The event will run **May 29-June 12, 2024**, and hackers have the opportunity to win cash bounties! unitaryHACK is unique to many quantum hackathon-style events for two big reasons:
+
+- Hackers support existing quantum computing projects. 
+- Hackers get paid for their work while building professional skills such as working on open source projects, and contributing to create a more functional, and featured, quantum computing stack.
+
+For more information, read our hacker guide and check out what happened in [unitaryHACK 2023](https://unitary.fund/posts/unitaryhack_wrap/) and get connected on our [Discord server](http://discord.unitary.fund)!
```
Comment (nathanshammah) [inline src/content/blog/2024_unitaryhack.md:14]:
```suggestion
unitaryHACK 2024 is the largest edition yet, hosting [**50 amazing open-source projects**](https://unitaryhack.dev/projects/): AI-inspired Classification of Quantum Computers, Amazon Braket Default Simulator, Amazon Braket SDK, AutoQASM, Azure Quantum Development Kit, bloqade-python, BQSKit, Braket.jl, BraketAHS.jl, BraketSimulator.jl, Cirq, Cirq Classiq Library, CUDA-Q, Fusion Blossom, graphix, HierarQcal, Ion(Q) Thruster, KQCircuits, lambeq, Metriq, Mitiq, OpenQAOA, OpenQASM 3 Parser, PennyLane, PennyLane Plugin for Amazon Braket, Perceval, Piccolo.jl, PyClifford, Qadence, qBraid-QIR, qBraid-SDK, Qiskit, Qiskit Aer, Qiskit Provider for Amazon Braket, qlasskit, Qrack, Quantum Machines QUA-to-Qiskit compiler and simulator, Quantum Open Source Foundation (QOSF), Quantum Universal Education, QuantumToolbox.jl, Qublitz, QuTiP Tutorials, rustworkx, scqubits, Tangelo
TensorCircuit
toqito
TorchQuantum

For more information, read our hacker guide and check out what happened in [unitaryHACK 2023](https://unitary.fund/posts/unitaryhack_wrap/) and get connected on our [Discord server](http://discord.unitary.fund)!
```

**Review by nathanshammah — APPROVED** [review]
Thanks @KallieFerguson. Looks good. Made small suggestions. Could add uH logo

**Pull request merged**